app/theme_defaults/_sub_fields.twig

```diff
@@ -9,89 +11,105 @@
+        {# Skip over the fields that are used in the slug, unless explicitly told not to,
+           using the `skip_uses` parameter. #}
+        {% if (record.contenttype.fields.slug.uses|default(null) is iterable) and skip_uses|default(true) %}
+            {% set omittedkeys = omittedkeys|merge(record.contenttype.fields.slug.uses) %}
+        {% endif %}
 
-{# Skip over the fields that are used in the slug, unless explicitly told not to,
-   using the `skip_uses` parameter. #}
-{% if (record.contenttype.fields.slug.uses|default(null) is iterable) and skip_uses|default(true) %}
-    {% set omittedkeys = omittedkeys|merge(record.contenttype.fields.slug.uses) %}
-{% endif %}
+        {# We also skip over the fields that are explicitly excluded. #}
+        {% if exclude|default is iterable %}
+            {% set omittedkeys = omittedkeys|merge(exclude) %}
+        {% endif %}
+    {% endspaceless -%}
 
-{# We also skip over the fields that are explicitly excluded. #}
-{% if exclude|default is iterable %}
-    {% set omittedkeys = omittedkeys|merge(exclude) %}
-{% endif %}
+    {# SECTION 2: LOOPING AND ITERATION - The actual looping is done here. #}
+    {% for key, value in record.values if (key not in omittedkeys) %}
 
-{# SECTION 2: LOOPING AND ITERATION #}
+        {# Is the field allowed to contain Twig functions #}
+        {% set allowtwig = record.contenttype.fields[key].allowtwig|default(false) %}
 
-{# The actual looping is done here. #}
-{% for key, value in record.values if (key not in omittedkeys) %}
+        {# Fields that are considered 'common': 'html', 'markdown', 'textarea',
+           'text', 'image', 'video' and 'imagelist' #}
+        {% if common|default(true) and (key not in exclude|default([])) %}
 
-    {# Fields that are considered 'common': 'html', 'markdown', 'textarea',
-       'text', 'image', 'video' and 'imagelist' #}
-    {% if common|default(true) %}
+            {% set fieldtype = record.fieldtype(key) %}
+            {% set value = record.get(key) %}
+            {{ block('common_fields') }}
 
-        {% set fieldtype = record.fieldtype(key) %}
-        {{ block('common_fields') }}
+        {% endif %}
 
-    {% endif %}
+        {# The rest of the built-in fieldtypes #}
+        {% if extended|default(false) and (key not in exclude|default([])) %}
 
-    {# The rest of the built-in fieldtypes #}
-    {% if extended|default(false) %}
+            {% set fieldtype = record.fieldtype(key) %}
+            {{ block('extended_fields') }}
 
-        {% set fieldtype = record.fieldtype(key) %}
-        {{ block('extended_fields') }}
+        {% endif %}
 
-    {% endif %}
+        {# Finally, the repeaters #}
+        {% if repeaters|default(false) == true and record.fieldtype(key) == "repeater" %}
 
-    {# Finally, the repeaters #}
-    {% if repeaters|default(true) and record.fieldtype(key) == "repeater" %}
+            {% if (global.request.get('_route') != "preview") %}
```

Comment:
Case in point :wink: 

Fixed and force pushing, thank you so much for finding time today mate!